skimage/util/compare.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+from ..util import img_as_float64
+from itertools import product
+
+
+def compare_images(image1, image2, method='diff'):
+    """
+    Return an image showing the differences between two images.
+
+    Parameters
+    ----------
+    image1, image2 : 2-D array
+        Images to process, must be of the same shape.
+    method : string, optional
+        Method used for the comparison.
+        Valid values are {'diff', 'blend', 'checkerboard'}
+
+    Returns
+    -------
+    comparison : 2-D array
+        Image showing the differences.
+    """
+    img1 = img_as_float64(image1)
+    img2 = img_as_float64(image2)
+
+    if method == 'diff':
+        comparison = np.abs(img2 - img1)
+    elif method == 'blend':
+        comparison = 0.5 * (img2 + img1)
+    elif method == 'checkerboard':
+        num_box = 9
```

Comment:
This would be useful as an optional parameter, wouldn't it?

Reply:
Yes. Here, there is room for discussions. Int, tuple of ints to specify 2 different dimensions...